### DISCUSSION TITLE
Shortcuts in slashcommands?

### DISCUSSION THREAD

**Discussion by isrbaral** [arnoudkooi/SN-Utils#283] [Q&A] [answered]
https://github.com/arnoudkooi/ServiceNow-Utils/blob/f8fd6fd437ac9dc1237f70091f8c619ac7e0cf96/inject.js#L630

Hi, Big fan of your extension. 

I was browsing your code and noticed this line, does it imply there's a way for me to create a shortcut to my pdi? I can never remember the 5 digit number and who wants to type that all out anyway, this line seems to imply there's some secret functionality that will let me automatically reference my pdi instead of having to tediously type out the who env name, is that true?

**Comment by arnoudkooi** [comment, marked as answer]
Yes if you create a slashcommans with url: `//env dev12345` it should open the current page in your pdi
